src/theme/theme.tsx

```diff
@@ -0,0 +1,45 @@
+// src/theme.tsx
+import { createTheme } from '@mui/material';
+
+const theme = createTheme({
```

Comment:
check the colors in the figma you can see text/high emphasis that should be included in the palette:{text:{high:color,low:color,medium:color}} check theme in material ui website